test/lit/exec/fuzzing-api.wast

```diff
@@ -13,8 +13,13 @@
  (import "fuzzing-support" "table-set" (func $table.set (param i32 funcref)))
  (import "fuzzing-support" "table-get" (func $table.get (param i32) (result funcref)))
 
+ (import "fuzzing-support" "call-export" (func $call.export (param i32)))
+ (import "fuzzing-support" "call-export-catch" (func $call.export.catch (param i32) (result i32)))
+
  (table $table 10 20 funcref)
 
+ ;; Note that the exported table appears first here, but in the binary and in
+ ;; the IR it is actually last, as we always add function exports first.
```

Comment:
I think we have a TODO in the text parser to fix this and add the exports to the IR in their original order. I guess we'll just have to update this test if we ever fix that.